crates/modules/rules-engine/README.md

```diff
@@ -3,18 +3,44 @@
 This module will check every Pulsar event against the configured set of rules.
 When a match is found, a threat event is generated.
 
-## Example rule
+## Rules
+
+Default rules folder is `/var/lib/pulsar/rules`. Each rule file is a `yaml` file containing a list of rules with the following required fields:
+
+- `name`: a unique name for identifying the rule
+- `type`: the type of event to match (e.g. `FileOpened`, `Exec`, `NetworkConnection`)
+- `description`: a human-readable description of the rule
+- `severity`: the event severity (e.g. `low`, `medium`, `high`, `critical`)
+- `category`: the threat category following the [MITRE ATT&CK](https://attack.mitre.org/) framework (e.g. `persistence`, `credential_access`, `defense_evasion`)
+- `condition`: a condition to match the event.
+
+Valid `severity` values are `low`, `medium`, `high`, and `critical`. 
+
+Valid `category` values are (use `generic` if the rule does not fit any of the following):
+`command_and_control`, `credential_access`, `defense_evasion`, `discovery`, `execution`, `exfiltration`, `impact`, `initial_access`, `lateral_movement`, `persistence`, `privilege_escalation`, `reconnaissance`, `resource_development`.
+
+### Examples
 
 Create a `/var/lib/pulsar/rules/example_rules1.yaml` with the following content:
 
+
 ```yaml
 - name: Read sensitive file from untrusted process
   type: FileOpened
   condition: header.image != "/usr/bin/sshd" && payload.filename == "/etc/shadow"
+  severity: high  
```

Comment:
Not sure if that should be "high" and, in general, whether we should have this rule at all. If we keep it, I would rather mark it as "low" or "medium".

I personally find this rule quite annoying, as it gets triggered any time you use `sudo`. That includes not only `sudo` typed manually in terminal, but also scripts. I think it makes more noise and it's much helpful with mitigating real attacks. @banditopazzo I remember that you complained about this rule too. :sweat_smile:

Reply:
This is just an example in the README, the final rule in `basic-rules` is `medium`